input/otlp/metadata.go

```diff
@@ -580,3 +585,19 @@ func replaceReservedRune(disallowedRunes string) func(r rune) rune {
 		return unicode.ToLower(r)
 	}
 }
+
+// NOTE: This will overwrite user fields set by `user.name=process.owner` from `otlp/metadata.go:translateResourceMetadata`.
+func addUserFields(attrKey string, attrVal pcommon.Value, updateEvent *modelpb.APMEvent) {
```

Comment:
```suggestion
func addUserFields(attrKey string, attrVal pcommon.Value, event *modelpb.APMEvent) {
```

nit: to be consistent with the rest of the code, `event` or `out` seems to be a better variable name